### PR TITLE
Fixed comment status text

### PR DESCRIPTION
**Pull request by BartoszJarocki**
Related issue: https://github.com/ParabolInc/parabol/issues/5097 

The main problem was with ids comparison.

### CODE REVIEW

File: packages/client/hooks/useBeforeUnload.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import {useRef, useEffect} from 'react'
+
+export const useBeforeUnload = (callback: () => any) => {
```
Comment (mattkrick) [inline packages/client/hooks/useBeforeUnload.ts:3]: oooo i like where this is headed!
this solution works right now, but what if in the future discussionId changes from "foo" to "bar"? the callback is persisted & points to the old value of discussionId. That'd be a tricky bug to track down.

easy fix:

`const cb = useEventCallback(callback)`

then you can pass cb to window event listeners

Comment (BartoszJarocki) [reply packages/client/hooks/useBeforeUnload.ts:3]: oh, that's 100% right 🤦

**Comment by BartoszJarocki** [comment]
@mattkrick fixed, I'd appreciate if you could take a look 🙏

**Comment by mattkrick** [comment]
:ok_hand:

**Pull request merged**